spec/tasks/combine_schema_spec.rb

```diff
@@ -168,4 +169,48 @@
       expect(actual.schema).to eq(expected.schema)
     end
   end
+
+  context "message queue schema" do
+    let(:output_filename) { File.join("dist", "message_queue.json") }
+    let(:details) { build_schema('details.json', properties: build_string_properties('detail')) }
+    let(:publisher_schema_dir) { project_root.join("formats", format_name, "publisher") }
+
+    around do |example|
+      format_root = ENV["FORMAT_ROOT"]
+      FileUtils.mv("dist/message_queue.json", "dist/message_queue.json.bkp")
```

Comment:
This seems a bit fragile: is there a possibility that the spec could crash halfway through, leaving the files in the wrong place?